Added pandas dataframe term about reset_index() method.

Sc1entifik opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

<!-- Please write a summary of the change, such as which topic(s) and file(s) that you have edited or created. Please also include relevant motivation and context: -->

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

<!-- Please check ALL the boxes: -->

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Codecademy/docs?pullRequest=1753) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=1753) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Codecademy/docs?pullRequest=1753) it.</sub>

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+
+### What Is .reset_index() 
+Sometimes there is a need to modify a dataframe object to better meet a certain use case. When these modifications are applied they often leave smaller dataframes with indexes that are messy and not continuous.
+Resetting the index with .reset_index() fixes these issues.
```
```suggestion
Resetting the index with **.reset_index()** fixes these issues.
```
We bold the first occurrence of the method.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+This article is about how the **.reset_index()** is used in pandas dataframes.
+
+
+## Syntax
```
```suggestion
## Syntax

```
There should be one line break before and after headers.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+Sometimes there is a need to modify a dataframe object to better meet a certain use case. When these modifications are applied they often leave smaller dataframes with indexes that are messy and not continuous.
+Resetting the index with .reset_index() fixes these issues.
+
+
```
```suggestion
```
remove extra line break. There should only be one line break before and after headers.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+Resetting the index with .reset_index() fixes these issues.
+
+
+### Actions Which Cause Indexing Issues 
```
```suggestion
## Actions Which Cause Indexing Issues 
```
Should be h2 and have one line break before and after.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+
+- Changing the order of columns.
+- Using a column or multiple columns of the database as an index instead of the default index pandas provides and then later needing a numbered index.
+- Filtering out certain rows of the dataframe based on the value of a column of data.
```
```suggestion
- Filtering out certain rows of the DataFrame based on the value of a column of data.
```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+- Changing the order of rows of data by sorting those rows by the values of a certain column of data such as Date or Employee ID Number.
+- Adding columns and or rows of data to the dataframe that were not present in the original dataframe.
+
+
```
```suggestion
```
Remove extra line break.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+- Adding columns and or rows of data to the dataframe that were not present in the original dataframe.
+
+
+### .reset_index() Parameters
```
```suggestion
### `.reset_index()` Parameters

```
Add line break after header.

Also, move this to the Syntax section. Parameters should be explained, and shown, in the syntax section.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
````diff
@@ -0,0 +1,168 @@
+import pandas as pd
+df.read_csv('some_csv.csv', index_col = 'Account Number')
+df.reset_index()
+```
````
There should be a paragraph or two after the syntax block explaining the parameters.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+
+
+### .reset_index() Parameters
+The .reset_index() method provides many parameters which allow for more precision when resetting a dataframe index. The following parameters will be used in the examples later in this article.
```
```suggestion
The .reset_index() method provides many parameters which allow for more precision when resetting a DataFrame index. The following parameters will be used in the examples later in this article.
```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+### .reset_index() Parameters
+The .reset_index() method provides many parameters which allow for more precision when resetting a dataframe index. The following parameters will be used in the examples later in this article.
+
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous dataframe index with the new index provided by .reset_index() otherwise it sets the new index in front of the old index.
```
```suggestion
- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous DataFram index with the new index provided by `.reset_index()` otherwise it sets the new index in front of the old index.
```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+The .reset_index() method provides many parameters which allow for more precision when resetting a dataframe index. The following parameters will be used in the examples later in this article.
+
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous dataframe index with the new index provided by .reset_index() otherwise it sets the new index in front of the old index.
+- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes .reset_index() makes to the current instance of the dataframe otherwise it creates a new dataframe instance with the changes applied to that dataframe.
```
```suggestion
- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes `.reset_index()` makes to the current instance of the DataFrame otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous dataframe index with the new index provided by .reset_index() otherwise it sets the new index in front of the old index.
+- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes .reset_index() makes to the current instance of the dataframe otherwise it creates a new dataframe instance with the changes applied to that dataframe.
+
+
```
```suggestion
```
Delete extra line break.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes .reset_index() makes to the current instance of the dataframe otherwise it creates a new dataframe instance with the changes applied to that dataframe.
+
+
+## Examples
```
```suggestion
## Examples

```
Add line break after header.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+In the following examples imagine working on a project about dogs and wanting to use data from an animal shelter about dogs in that shelter but the data in the .csv file has data about cats and dogs.
+To follow along a copy of the [Austin_Animal_Center_intakes.csv](https://www.kaggle.com/datasets/jackdaoud/animal-shelter-analytics) can be downloaded from Kaggle.
+
+## Example Of The Original Dataframe
```
```suggestion
### Example Of The Original DataFrame

```
h3, line break after header.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
````diff
@@ -0,0 +1,168 @@
+4  Doberman Pinsch/Australian Cattle Dog      Tan/Gray
+```
+
+## Example Removing Cats From Dataframe
````
```suggestion
### Example Removing Cats From DataFrame
```
h3

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
````diff
@@ -0,0 +1,168 @@
+print(df)
+```
+
+## Output
````
```suggestion
This results in the following output:

```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
````diff
@@ -0,0 +1,168 @@
+print(df)
+```
+
+## Output without df.reset_index(inplace = True, drop = True)
````
```suggestion
This is the output without `df.reset_index(inplace = True, drop = True)`:

```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+The indexing now jumps from two to four after the row containing the cat is removed. This can become very messy when dealing with large dataframes containing hundreds or even thousands of rows.
+
+
+## Output With df.reset_index(inplace = True, drop = True)
```
```suggestion
This is the output with `df.reset_index(inplace = True, drop = True)`:

```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
````diff
@@ -0,0 +1,168 @@
+```
+
+After applying df.reset_index(inplace = True, drop = True) the dataframe index order is now neat and continuous for easy indexing.
+Sorry, Johnny Ringo dogs are just cooler than cats!
````
```suggestion
```
Let's delete this last line.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
````diff
@@ -0,0 +1,168 @@
+3  Doberman Pinsch/Australian Cattle Dog     Tan/Gray
+```
+
+After applying df.reset_index(inplace = True, drop = True) the dataframe index order is now neat and continuous for easy indexing.
````
```suggestion
After applying `df.reset_index(inplace = True, drop = True)` the DataFrame index order is now neat and continuous for easy indexing.
```

SSwiniarski reviewed (changes requested):
@Sc1entifik, I did a first review and left a bunch of comments. One important one is to remove the two extra files you have in the terms/resetindex folder. The rest is mostly formatting and reorganization to bring this in line with our preferred style for Docs entries.

Sc1entifik commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,168 @@
+---
+Title: '.reset_index()'
+Description: 'This article is about how the .reset_index() is used in pandas dataframes.'
```
Hi I made all the changes you requested from this article and pushed it again. This is my first time contributing to someone else's project with Git Hub so I am a little green. If I did it wrong or I need to make a new pull request or something after I made these changes and pushed them again please let me know.

Sc1entifik commented on content/pandas/concepts/dataframe/terms/resetindex/.~lock.Austin_Animal_Center_Intakes.csv#:
```diff
@@ -0,0 +1 @@
+,scientifik,lincomp1,28.02.2023 11:52,file:///home/scientifik/.config/libreoffice/4;
```
I believe I had that pdf opened in libreoffice as a spreadsheet when I committed my repository the first time which is why I think .lock.Austin_Animal_Center_Intakes.csv showed up in the first place. I removed the pdf from my resetindex folder so I think that should remove both of those files. If that doesn't remove them please let me know what I need to do to get rid of it because it doesn't show up as a file on my end anymore. I also made all the changes you suggested then pushed to the repository again.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,165 @@
+CatalogContent:
+  - 'learn-python'
+  - 'paths/data-science'
+---
```
```suggestion
---

```
Need a line break after the metadata.

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,165 @@
+
+The .reset_index() method provides many parameters which allow for more precision when resetting a DataFrame index. The following parameters will be used in the examples later in this article.
+
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous DataFrame index with the new index provided by .reset_index() otherwise it sets the new index in front of the old index.
```
```suggestion
- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous DataFrame index with the new index provided by `.reset_index()` otherwise it sets the new index in front of the old index.
```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
```diff
@@ -0,0 +1,165 @@
+The .reset_index() method provides many parameters which allow for more precision when resetting a DataFrame index. The following parameters will be used in the examples later in this article.
+
+- *drop:* The drop parameter can be set to True or False and is set to False by default. When this parameter is set to True it replaces the previous DataFrame index with the new index provided by .reset_index() otherwise it sets the new index in front of the old index.
+- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes .reset_index() makes to the current instance of the DataFrame otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
```
```suggestion
- *inplace:* The inplace parameter can be set to True or False and is set to False by default. When this parameter is set to True it applies all changes `.reset_index()` makes to the current instance of the DataFrame otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
```

SSwiniarski commented on content/pandas/concepts/dataframe/terms/resetindex/reset-index.md:
````diff
@@ -0,0 +1,165 @@
+print(df)
+```
+
+## Output without `df.reset_index(inplace = True, drop = True)`:
````
```suggestion
This is the output without `df.reset_index(inplace = True, drop = True)`:
```

SSwiniarski reviewed (changes requested):
@Sc1entifik I left a few more comments. Mostly formatting issues.

Sc1entifik commented:
Hi. This is my first pull request I made on someone else's project. I think I did everything that I needed to via your requests but I just want to make sure I didn't miss something somewhere.

Sc1entifik commented:
Ok I made these changes. I'll request another review. Thanks.

SSwiniarski reviewed (approved):
@Sc1entifik looks good for a second review.

Sc1entifik commented:
Ok Thanks. I'll be checking in on it from time to time. Let me know if there is anything else I need to do.

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+- Changing the order of rows of data by sorting those rows by the values of a certain column of data such as Date or Employee ID Number.
+- Adding columns and or rows of data to the DataFrame that were not present in the original DataFrame.
+
+## Examples
```
```suggestion
## Example
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+
+## Examples
+
+In the following examples imagine working on a project about dogs and wanting to use data from an animal shelter about dogs in that shelter but the data in the .csv file has data about cats and dogs.
```
```suggestion

```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+## Examples
+
+In the following examples imagine working on a project about dogs and wanting to use data from an animal shelter about dogs in that shelter but the data in the .csv file has data about cats and dogs.
+To follow along a copy of the [Austin_Animal_Center_intakes.csv](https://www.kaggle.com/datasets/jackdaoud/animal-shelter-analytics) can be downloaded from Kaggle.
```
```suggestion
To follow along a copy of the [Austin_Animal_Center_intakes.csv](https://data.austintexas.gov/Health-and-Community-Services/Austin-Animal-Center-Intakes/wter-evkm) can be downloaded from city of Austin data portal.
```

caupolicandiaz replied:
This is a small change, but ideally we link back to a source that has an explicit (public) license. In this case Kaggle is an intermediary and the source of the data presented on this page could change.

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+In the following examples imagine working on a project about dogs and wanting to use data from an animal shelter about dogs in that shelter but the data in the .csv file has data about cats and dogs.
+To follow along a copy of the [Austin_Animal_Center_intakes.csv](https://www.kaggle.com/datasets/jackdaoud/animal-shelter-analytics) can be downloaded from Kaggle.
+
+### Example Of The Original DataFrame
```
```suggestion
### The Original DataFrame
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
````diff
@@ -0,0 +1,167 @@
+4  Doberman Pinsch/Australian Cattle Dog      Tan/Gray
+```
+
+### Example Removing Cats From Dataframe
````
```suggestion
### Removing Cats From the Dataframe
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
````diff
@@ -0,0 +1,167 @@
+### Example Removing Cats From Dataframe
+
+```py
+import pandas as pd
+
+df = pd.read_csv('Austin_Animal_Center_intakes.csv').head()
+pd.set_option('display.max_columns', None)
````
```suggestion

```

caupolicandiaz replied:
I don't think we need to include these lines in both blocks

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+df = pd.read_csv('Austin_Animal_Center_intakes.csv').head()
+pd.set_option('display.max_columns', None)
+
+#this section of code removes the furball from our dog DataFrame
```
```suggestion
# This section of code removes the furball from our dog DataFrame
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+#this section of code removes the furball from our dog DataFrame
+df = df[df['Animal Type'] != 'Cat']
+
+#uncommenting the line below this line will remove the index of the original DataFrame and reset the order
```
```suggestion
# Uncommenting the line below this line will remove the index of the original DataFrame and reset the order
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+df = df[df['Animal Type'] != 'Cat']
+
+#uncommenting the line below this line will remove the index of the original DataFrame and reset the order
+#df.reset_index(inplace = True, drop = True)
```
```suggestion
# df.reset_index(inplace = True, drop = True)
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,167 @@
+
+The indexing now jumps from two to four after the row containing the cat is removed. This can become very messy when dealing with large DataFrames containing hundreds or even thousands of rows.
+
+This is the output with `df.reset_index(inplace = True, drop = True)`:
```
```suggestion
This is the output with `df.reset_index(inplace = True, drop = True)`:

```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
````diff
@@ -0,0 +1,167 @@
+This is the output without `df.reset_index(inplace = True, drop = True)`:
+
+```shell
+
````
```suggestion
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
````diff
@@ -0,0 +1,167 @@
+This results in the following output:
+
+```shell
+
````
```suggestion
```

caupolicandiaz reviewed (changes requested):
@Sc1entifik I've completed my initial review. Please update and tag me when you've finished making changes. Thanks

Sc1entifik commented:
I made all the suggested changes.

caupolicandiaz commented:
> I made all the suggested changes.

This isn't complete, please review my comments again. There are several that have not been addressed.

Sc1entifik commented:
> > I made all the suggested changes.
> 
> This isn't complete, please review my comments again. There are several that have not been addressed.

This is my first time doing a pull request and using the Git Hub website UI for this. I think I found them all this time.

caupolicandiaz commented:
> > > I made all the suggested changes.
> > 
> > 
> > This isn't complete, please review my comments again. There are several that have not been addressed.
> 
> This is my first time doing a pull request and using the Git Hub website UI for this. I think I found them all this time.

No worries, we all make mistakes. There are still edits missing:
- All parameters should be included in the syntax section (currently you only cover 2)
- Looks like there are missing backticks required to close the syntax block
- There are additional lines after this section ( there should never really be more than a single empty line between content )

Sc1entifik commented:
> > > > I made all the suggested changes.
> > > 
> > > 
> > > This isn't complete, please review my comments again. There are several that have not been addressed.
> > 
> > 
> > This is my first time doing a pull request and using the Git Hub website UI for this. I think I found them all this time.
> 
> No worries, we all make mistakes. There are still edits missing:
> 
> * All parameters should be included in the syntax section (currently you only cover 2)
> * Looks like there are missing backticks required to close the syntax block
> * There are additional lines after this section ( there should never really be more than a single empty line between content )



> > > > I made all the suggested changes.
> > > 
> > > 
> > > This isn't complete, please review my comments again. There are several that have not been addressed.
> > 
> > 
> > This is my first time doing a pull request and using the Git Hub website UI for this. I think I found them all this time.
> 
> No worries, we all make mistakes. There are still edits missing:
> 
> * All parameters should be included in the syntax section (currently you only cover 2)
> * Looks like there are missing backticks required to close the syntax block
> * There are additional lines after this section ( there should never really be more than a single empty line between content )

I assume you want all parameters and not just the ones used in the example? I added those and fixed the other issues mentioned with my newest push.

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,160 @@
+- *col_level:* Takes integer or string values and is set to 0 by default. Determines what level the labels are inserted into when the columns have multiple levels. The first level is set by default.
+- *col_fill:* Takes a string, list, or None and is set to `None` by default. Determines how the other levels of are named when the columns have multiple levels. Uses the index name by default.
+- *allow_duplicates:* Optional parameter which takes a boolean value and is set to `lib.no_default` by default. When this parameter is set to `True` it allows duplicate column labels to be created.
+- *names:* Takes interger, string, 1-dimensional list, or None values and is set to `None` by default. Renames the index DataFrame column. In the case that the Dataframe has a MultiIndex this value has to be a list or tuple equal in length to the number of levels.
```
```suggestion
- *names:* Takes integer, string, 1-dimensional list, or None values and is set to None by default. Renames the index DataFrame column. In the case that the Dataframe has a MultiIndex this value has to be a list or tuple equal in length to the number of levels.
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,160 @@
+- *inplace:* Takes a boolean value and is set to `False` by default. When this parameter is set to `True` it applies all changes to the current instance of the DataFrame, otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
+- *level:* Takes integer, string, tuple, list, or None values and is set to `None` by default. Removes the levels provided from the index.
+- *col_level:* Takes integer or string values and is set to 0 by default. Determines what level the labels are inserted into when the columns have multiple levels. The first level is set by default.
+- *col_fill:* Takes a string, list, or None and is set to `None` by default. Determines how the other levels of are named when the columns have multiple levels. Uses the index name by default.
```
```suggestion
- *col_fill:* Takes a string, list, or None and is set to None by default. Determines how the other levels are named when the columns have multiple levels. Uses the index name by default.
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
```diff
@@ -0,0 +1,160 @@
+
+- *drop:* Takes a boolean value and is set to `False` by default. When this parameter is set to `True` it replaces the previous DataFrame index with the new index provided by `.reset_index()`, otherwise it sets the new index in front of the old index.
+- *inplace:* Takes a boolean value and is set to `False` by default. When this parameter is set to `True` it applies all changes to the current instance of the DataFrame, otherwise it creates a new DataFrame instance with the changes applied to that DataFrame.
+- *level:* Takes integer, string, tuple, list, or None values and is set to `None` by default. Removes the levels provided from the index.
```
```suggestion
- *level:* Takes integer, string, tuple, list, or None values and is set to None by default. Removes the given levels from the index, by default all levels are removed.
```

caupolicandiaz commented on content/pandas/concepts/dataframe/terms/reset-index/reset-index.md:
````diff
@@ -0,0 +1,160 @@
+df = dataframe_value.reset_index()
+```
+
+The `.reset_index()` method provides the following parameters:
````
Please reorder the parameters to match: level, drop, inplace, col_level, col_fill, allow_duplicates, names

caupolicandiaz reviewed (changes requested):
@Sc1entifik Just a few more edits. Tag me when you're finished. Thanks

Sc1entifik commented:
@caupolicandiaz Ok it's been done.

caupolicandiaz reviewed (approved):
@Sc1entifik Looks good.

Sc1entifik commented:
> @Sc1entifik Looks good.

Ok thanks. Do I need to do anything from here like update the branch? Or do I just wait and check back in every now and then?

SSwiniarski commented:
@Sc1entifik congrats on the new entry. The live entry is here: https://www.codecademy.com/resources/docs/pandas/dataframe/reset-index You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
![image](https://user-images.githubusercontent.com/86081858/224488562-431bb9e4-3687-4679-957f-e72a220e0d47.png)

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

![image](https://user-images.githubusercontent.com/86081858/224488591-e4eab222-7912-4fc3-a767-225b1f5b41b4.png)

2.) Then log out and log back into Codecademy using your GitHub login:

![image](https://user-images.githubusercontent.com/86081858/224488619-1b861f46-55ea-4d12-a069-057282dafae1.png)

Sc1entifik commented:
> @Sc1entifik congrats on the new entry. The live entry is here: https://www.codecademy.com/resources/docs/pandas/dataframe/reset-index You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do. ![image](https://user-images.githubusercontent.com/86081858/224488562-431bb9e4-3687-4679-957f-e72a220e0d47.png)
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> ![image](https://user-images.githubusercontent.com/86081858/224488591-e4eab222-7912-4fc3-a767-225b1f5b41b4.png)
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> ![image](https://user-images.githubusercontent.com/86081858/224488619-1b861f46-55ea-4d12-a069-057282dafae1.png)

Hello I followed the steps and logged into my CodeCademy through my Git Hub account but it seems I still am not getting credited for my contribution. Any help would be appreciated.

Sc1entifik commented:
> @Sc1entifik congrats on the new entry. The live entry is here: https://www.codecademy.com/resources/docs/pandas/dataframe/reset-index You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do. ![image](https://user-images.githubusercontent.com/86081858/224488562-431bb9e4-3687-4679-957f-e72a220e0d47.png)
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> ![image](https://user-images.githubusercontent.com/86081858/224488591-e4eab222-7912-4fc3-a767-225b1f5b41b4.png)
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> ![image](https://user-images.githubusercontent.com/86081858/224488619-1b861f46-55ea-4d12-a069-057282dafae1.png)

Never mind it is working now thanks.
